### ISSUE TITLE
Some CurseForge mods have null icons

### ISSUE THREAD

**Issue by douira** [theRookieCoder/ferium#41]
I can't add this Curseforge mod: https://www.curseforge.com/minecraft/mc-mods/status-effect-bars to a profile. Instead, it responds with the following:

```
% ferium add-curseforge 615459
Adding mod... error decoding response body: invalid type: null, expected struct ModAsset at line 1 column 553
```

This doesn't happen with any other Curseforge mods or Modrinth mods. Is this an issue with how the resources of this mod are structured or is Curseforge denying me access to this mod's metadata?

**Comment by theRookieCoder** [comment]
I've been having this same issue with Terralith, where the icon is `null` for some reason. I initially thought this was some niche problem with Terralith. I think curseforge is denying access because obviously the mods do have an icon as seen in the website.

**Comment by theRookieCoder** [comment]
Huh, now Terralith has its icon again. So it does seem to be an issues on CurseForge's side, but I'll make the `logo` nullable anyways so that no one runs into this issue again. The problem is the [CurseForge docs](https://docs.curseforge.com) don't tell you what's nullable and what's always there

**Comment by theRookieCoder** [comment]
Fixed by [Furse `1.1.1`](https://github.com/theRookieCoder/furse/commit/b5795c1bd6753465671a64dd4b5080ce62b35f73)
